Cargo.toml

```diff
@@ -17,14 +17,14 @@ anyhow = "1.0.75"
 async-trait = "0.1"
 clap = { version = "3.1.15", features = ["derive", "env"] }
 serde = "1.0"
-spin-app = { git = "https://github.com/fermyon/spin", tag = "v2.3.1" }
-spin-core = { git = "https://github.com/fermyon/spin", tag = "v2.3.1" }
-spin-trigger = { git = "https://github.com/fermyon/spin", tag = "v2.3.1" }
+spin-app = { git = "https://github.com/fermyon/spin" }
+spin-core = { git = "https://github.com/fermyon/spin" }
+spin-trigger = { git = "https://github.com/fermyon/spin" }
 tokio = { version = "1.23", features = ["full"] }
 tokio-cron-scheduler = "0.9.4"
```

Comment:
nit: remove the `tokio-cron-scheduler` and any other dependencies that aren't required here.